### PR TITLE
Add avif as a recognized image format

### LINKED ISSUE #240: Improve avif detection
Explicitly check for `.avif` and `image/avif` when defining the `type` and `format` fields

### PR DESCRIPTION
**Pull request by rviscomi**
Fixes #240

### CODE REVIEW

**Comment by tunetheweb** [comment]
Didn't include `heif`? Not sure how much it's used.

File: bulktest/batch_lib.inc [hunk 1]
```diff
@@ -963,7 +963,7 @@ function prettyType($mimeType, $ext) {
 	else if ( "eot" === $ext || "ttf" === $ext || "woff" === $ext || "woff2" === $ext || "otf" === $ext ) {
 		return "font";
 	}
-	else if ( "png" === $ext || "gif" === $ext || "jpg" === $ext || "jpeg" === $ext || "webp" === $ext || "ico" === $ext || "svg" === $ext ) {
+	else if ( "png" === $ext || "gif" === $ext || "jpg" === $ext || "jpeg" === $ext || "webp" === $ext || "ico" === $ext || "svg" === $ext || "avif" === $ext || "jxl" === $ext || "heic" === $ext ) {
```
Comment (tunetheweb) [inline bulktest/batch_lib.inc:966]:
```suggestion
	else if ( "png" === $ext || "gif" === $ext || "jpg" === $ext || "jpeg" === $ext || "webp" === $ext || "ico" === $ext || "svg" === $ext || "avif" === $ext || "jxl" === $ext || "heic" === $ext || "heif" === $ext ) {
```

File: bulktest/batch_lib.inc [hunk 2]
```diff
@@ -994,7 +994,7 @@ function prettyType($mimeType, $ext) {
 function getFormat($prettyType, $mimeType, $ext) {
 	if ( "image" === $prettyType ) {
 		// Order by most popular first.
-		foreach( array("jpg", "png", "gif", "webp", "svg", "ico") as $type) {
+		foreach( array("jpg", "png", "gif", "webp", "svg", "ico", "avif", "jxl", "heic") as $type) {
```
Comment (tunetheweb) [inline bulktest/batch_lib.inc:997]:
```suggestion
		foreach( array("jpg", "png", "gif", "webp", "svg", "ico", "avif", "jxl", "heic", "heif") as $type) {
```

**Review by tunetheweb — COMMENTED** [review]
Should we add `heif`?

**Comment by rviscomi** [comment]
Synced with the HA test server.

**Pull request merged**